docs/Building.md

```diff
@@ -118,42 +115,42 @@ and llvm.
     source utils/env_setup.sh <mlir-aie>/install <llvm dir>/install
     ```
 
+6. **Only for the VCK5000:** The PCIe AIR runtime requires the use of the [AIR PCIe kernel driver](https://github.com/Xilinx/ROCm-air-platforms/tree/main/driver). The driver directory in the [ROCm-air-platforms](https://github.com/Xilinx/ROCm-air-platforms) repository contains documentation on how to compile and load the AIR PCIe kernel driver.
```

Comment:
The second sentence seems not very useful as it points to the same directory. I'd say "This directory contains ...".

Reply:
Thanks for catching! Addressed in [4ce47c8](https://github.com/Xilinx/mlir-aie/pull/1597/commits/4ce47c857d36eb4a344600c0b16b175ee7792ffc).